modules/beam/src/test/resources/enrichments_wrong/foo.json

```diff
@@ -0,0 +1 @@
+bar
```

Comment:
Don't think we need it?

Reply:
It's used here : https://github.com/snowplow/enrich/pull/392/files#diff-5e7b8cfab5c73bb6092007871ff982d471bdb1b42ac80a82350ab64669d87715R95
It's not directly related to this issue but I took the opportunity to improve unit testing to our configuration parsing

Reply:
Ah ok, sorry - I somehow thouoght it's in the repo's root.